packages/scoutgame-ui/src/components/notifications/NotificationCard.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import type { ScoutAppNotification } from '@charmverse/core/prisma-client';
+import { Typography, Stack, Card, Link, Button } from '@mui/material';
+import { AppNotificationTypesRecord } from '@packages/scoutgame/notifications/appNotificationConstants';
+import { getShortenedRelativeTime } from '@packages/utils/dates';
+import { useAction } from 'next-safe-action/hooks';
+
+import { toggleAppNotification } from '../../actions/toggleAppNotification';
+import { useMdScreen } from '../../hooks/useMediaScreens';
+
+export function NotificationCard({ notification }: { notification: ScoutAppNotification }) {
+  const { execute, isExecuting } = useAction(toggleAppNotification);
+  const notificationType = notification.notificationType as keyof typeof AppNotificationTypesRecord;
+  const title = AppNotificationTypesRecord[notificationType].title;
+
+  const description =
```

Comment:
i counted only 3 types that don't use a function. might be good to just make them all functions so this code is simpler

Reply:
yeah good point